go.sum

```diff
@@ -84,6 +84,9 @@ collectd.org v0.5.0/go.mod h1:A/8DzQBkF6abtvrT2j/AU/4tiBgJWYyh0y/oB/4MlWE=
 contrib.go.opencensus.io/exporter/prometheus v0.3.0/go.mod h1:rpCPVQKhiyH8oomWgm34ZmgIdZa8OVYO5WAIygPbBBE=
 dmitri.shuralyov.com/gpu/mtl v0.0.0-20190408044501-666a987793e9/go.mod h1:H6x//7gZCb22OMCxBHrMx7a5I7Hp++hsVxbQ4BYO7hU=
 gioui.org v0.0.0-20210308172011-57750fc8a0a6/go.mod h1:RSH6KIUZ0p2xy5zHDxgAM4zumjgTw83q2ge/PI+yyw8=
+github.com/99designs/keyring v1.1.7-0.20211011120927-33891b18a1fa h1:3LyIWbLY03gktapLPoOMdtISxb6FyKMpWNUrjjEYT8Y=
+github.com/99designs/keyring v1.1.7-0.20211011120927-33891b18a1fa/go.mod h1:zTekYv1iRUODu1CH/T2GVzta6sJceVcTI7CWJ1FwNX0=
```

Comment:
Will look into those. However, this will require changes in the dependency projects that we do not control.

Reply:
IME, you don't need to drop these since they aren't very complicated. I just wanted to call out that if there were bugs in this code that affected telegraf, it is likely that we would have to patch them ourselves.